### PR TITLE
[chore] update CHANGELOG in preperation for v0.110 release

### PR DESCRIPTION
**Pull request by hughesjj**

### CODE REVIEW

**Review by crobert-1 — COMMENTED** [review]
We need to include changelog notes for any upstream change that is relevant and impactful to our end-users as well.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,10 +2,22 @@
...
+
 ### 💡 Enhancements 💡
 
 - (Splunk) Update instruction for partial discovery ([#5402](https://github.com/signalfx/splunk-otel-collector/pull/5402))
 
+### 🧰 Bug fixes 🧰
+
+- (Splunk) Choco upgrade should preserve collector service custom env vars ([#5386](https://github.com/signalfx/splunk-otel-collector/pull/5386)) 
+- (Splunk) Remove redundant discovery.receiver.rule attribute ([#5403](https://github.com/signalfx/splunk-otel-collector/pull/5403)) 
+- (Splunk) Remove redundant resource attributes ([#5409](https://github.com/signalfx/splunk-otel-collector/pull/5409))
```
Comment (dmitryax) [inline CHANGELOG.md:19]: I'd remove these. They are only applicable with enabled feature gate which was introduced a version ago and very unlikely was used by anyone

**Comment by crobert-1** [comment]
> Gotcha, is that just "copy everything relevant to components we use as-is" or should I workshop the inclusion/exclusion with Product?

I'd say start with copy, and then as part of the review we can remove anything we may not be interested in, or reword as necessary 👍

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,10 +2,81 @@
...
+- (Contrib) `signalfxexporter`: Ensure token is not sent through for event data ([#35154](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35154))
+- (Contrib) `prometheusreceiver`: Fix the retrieval of scrape configurations by also considering scrape config files ([#34786](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34786))
+- (Contrib) `redactionprocessor`: Fix panic when using the redaction processor in a logs pipeline ([#35331](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35331))
+- (Contrib) `exporter/splunkhec`: Fix incorrect claim that the exporter doesn't mutate data when batching is enabled. The bug lead to runtime panics when the exporter was used with the batcher enabled in a fanout scenario. ([#35306](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35306))
+
+
```
Comment (crobert-1) [inline CHANGELOG.md:79]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,10 +2,81 @@
...
+- (Contrib) `pkg/stanza`: Move filelog.container.removeOriginalTimeField feature gate to beta. Disable the filelog.container.removeOriginalTimeField feature gate to get the old behavior. ([#33389](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33389))
+- (Contrib) `resourcedetectionprocessor`: Move processor.resourcedetection.hostCPUSteppingAsString feature gate to stable. ([#31136](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/31136))
+- (Contrib) `resourcedetectionprocessor`: Remove processor.resourcedetection.hostCPUModelAndFamilyAsString feature gate. ([#29025](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/29025))
+
+
+### 🚩 Deprecations 🚩
+- (Core) `processorhelper`: deprecate accepted/refused/dropped metrics ([#11201](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/11201))
+- (Contrib) `hostmetricsreceiver`: Set the receiver.hostmetrics.normalizeProcessCPUUtilization feature gate to stable. ([#34763](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34763))
+
```
Comment (crobert-1) [inline CHANGELOG.md:23]:
```suggestion
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,10 +2,81 @@
...
+- (Core) processorhelper: Remove deprecated [Traces|Metrics|Logs]Inserted funcs ([#11151](https://github.com/open-telemetry/opentelemetry-collector/pull/11151))
+- (Core) config: Mark UseLocalHostAsDefaultHostfeatureGate as stable  ([#11235](https://github.com/open-telemetry/opentelemetry-collector/pull/11235))
+- (Contrib) `pkg/stanza`: Move filelog.container.removeOriginalTimeField feature gate to beta. Disable the filelog.container.removeOriginalTimeField feature gate to get the old behavior. ([#33389](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33389))
+- (Contrib) `resourcedetectionprocessor`: Move processor.resourcedetection.hostCPUSteppingAsString feature gate to stable. ([#31136](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/31136))
+- (Contrib) `resourcedetectionprocessor`: Remove processor.resourcedetection.hostCPUModelAndFamilyAsString feature gate. ([#29025](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/29025))
+
+
```
Comment (crobert-1) [inline CHANGELOG.md:19]:
```suggestion
```

**Review by crobert-1 — COMMENTED** [review]
Looking good! Some whitespace nits, and comments:

1. Any component needs to be formatted with the `formatting`, I see a couple missing.
2. We can remove any changelog entries for components not in our distro.

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,10 +2,81 @@
...
+- (Contrib) `headersetterextension`: adding default_value config. `default_value` config item applied in case context value is empty. ([#34412](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34412))
+- (Contrib) `kafkaexporter`: Add support for encoding extensions in the Kafka exporter. This change adds support for encoding extensions in the Kafka exporter. Loading extensions takes precedence over the internally supported encodings. ([#34384](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34384))
+- (Contrib) `datadogexporter`: Adds exporter.datadogexporter.metricremappingdisabled featuregate which disables renaming OpenTelemetry metrics to match Datadog semantics. This feature gate is only for internal use. ([#35025](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35025))
+- (Contrib) `otelarrowexporter`: Add BatcherConfig field following similar in OTLP exporter. ([#34802](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34802))
```
Comment (pjanotti) [inline CHANGELOG.md:44]: I don't think we include any of these exporters. The list should filter out all components that are not part of our build.

File: CHANGELOG.md [hunk 6]
```diff
@@ -2,10 +2,81 @@
 
 ## Unreleased
 
+## v0.110.0
+
+This Splunk OpenTelemetry Collector release includes changes from the [opentelemetry-collector v0.110.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.110.0) and the [opentelemetry-collector-contrib v0.110.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.110.0) releases where appropriate.
+
+Additionally, updates `splunk-otel-javaagent` to [`v2.8.1`](https://github.com/signalfx/splunk-otel-java/releases/tag/v2.8.1) and `jmx-metric-gatherer` to [`v1.39.0`](https://github.com/open-telemetry/opentelemetry-java-contrib/releases/tag/v1.39.0)
+
+### 🛑 Breaking changes 🛑
```
Comment (pjanotti) [inline CHANGELOG.md:11]:
```suggestion
### 🛑 Breaking changes 🛑

```

File: CHANGELOG.md [hunk 7]
```diff
@@ -2,10 +2,81 @@
...
+- (Core) processorhelper: Remove deprecated [Traces|Metrics|Logs]Inserted funcs ([#11151](https://github.com/open-telemetry/opentelemetry-collector/pull/11151))
+- (Core) config: Mark UseLocalHostAsDefaultHostfeatureGate as stable  ([#11235](https://github.com/open-telemetry/opentelemetry-collector/pull/11235))
+- (Contrib) `pkg/stanza`: Move filelog.container.removeOriginalTimeField feature gate to beta. Disable the filelog.container.removeOriginalTimeField feature gate to get the old behavior. ([#33389](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33389))
+- (Contrib) `resourcedetectionprocessor`: Move processor.resourcedetection.hostCPUSteppingAsString feature gate to stable. ([#31136](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/31136))
+- (Contrib) `resourcedetectionprocessor`: Remove processor.resourcedetection.hostCPUModelAndFamilyAsString feature gate. ([#29025](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/29025))
+
+
+### 🚩 Deprecations 🚩
```
Comment (pjanotti) [inline CHANGELOG.md:20]:
```suggestion
### 🚩 Deprecations 🚩

```

File: CHANGELOG.md [hunk 8]
```diff
@@ -2,10 +2,81 @@
...
+- (Core) `processorhelper`: deprecate accepted/refused/dropped metrics ([#11201](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/11201))
+- (Contrib) `hostmetricsreceiver`: Set the receiver.hostmetrics.normalizeProcessCPUUtilization feature gate to stable. ([#34763](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34763))
+
+
 ### 💡 Enhancements 💡
 
 - (Splunk) Update instruction for partial discovery ([#5402](https://github.com/signalfx/splunk-otel-collector/pull/5402))
```
Comment (pjanotti) [inline CHANGELOG.md:27]: nit: IIRC we typically `Splunk` bullets bellow `Core` and `Contrib` ones.

File: CHANGELOG.md [hunk 9]
```diff
@@ -2,10 +2,81 @@
...
+
+- (Core) `service`: Ensure process telemetry is registered when internal telemetry is configured with readers instead of an address. ([#11093](https://github.com/open-telemetry/opentelemetry-collector/pull/11093))
+- (Core) `otelcol`: Fix the Windows Event Log configuration when running the Collector as a Windows service. ([#5297](https://github.com/open-telemetry/opentelemetry-collector/pull/5297), [#11051](https://github.com/open-telemetry/opentelemetry-collector/pull/11051))
+- (Core) `builder`: Honor build_tags in config ([#11156](https://github.com/open-telemetry/opentelemetry-collector/pull/11156))
+- (Core) `builder`: Fix version for providers in the default config ([#11123](https://github.com/open-telemetry/opentelemetry-collector/pull/11123))
+- (Core) `confmap`: Fix loading config of a component from a different source.  This issue only affected loading the whole component config, loading parts of a component config from a different source was working correctly. ([#11154](https://github.com/open-telemetry/opentelemetry-collector/pull/11154))
```
Comment (pjanotti) [inline CHANGELOG.md:66]: These don't affect our users and shouldn't be included.

File: CHANGELOG.md [hunk 10]
```diff
@@ -2,10 +2,81 @@
...
+- (Contrib) `hostmetricsreceiver`: Set the receiver.hostmetrics.normalizeProcessCPUUtilization feature gate to stable. ([#34763](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34763))
+
+
 ### 💡 Enhancements 💡
 
 - (Splunk) Update instruction for partial discovery ([#5402](https://github.com/signalfx/splunk-otel-collector/pull/5402))
 
+- (Core) `pdata`: Add support to MoveTo for Map, allow avoiding copies ([#11175](https://github.com/open-telemetry/opentelemetry-collector/pull/11175))
+- (Core) `mdatagen`: Add stability field to telemetry metrics, allowing the generated description to include a stability string. ([#11160](https://github.com/open-telemetry/opentelemetry-collector/pull/11160))
+- (Core) `confignet`: Mark module as Stable. ([#9801](https://github.com/open-telemetry/opentelemetry-collector/pull/9801))
```
Comment (pjanotti) [inline CHANGELOG.md:31]: At first glance I don't think that these 3 affect our users.

**Review by pjanotti — APPROVED** [review]

File: CHANGELOG.md [hunk 11]
```diff
@@ -2,10 +2,62 @@
...
+- (Contrib) `sapmexporter`: Prioritize token in context when accesstokenpassthrough is enabled ([#35123](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35123))
+- (Contrib) `tailsamplingprocessor`: Fix the behavior for numeric tag filters with inverse_match set to true. ([#34296](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34296))
 - (Splunk) Update instruction for partial discovery ([#5402](https://github.com/signalfx/splunk-otel-collector/pull/5402))
 
+### 🧰 Bug fixes 🧰
+
+- (Core) `service`: Ensure process telemetry is registered when internal telemetry is configured with readers instead of an address. ([#11093](https://github.com/open-telemetry/opentelemetry-collector/pull/11093))
+- (Contrib) `splunkenterprise`: Fix a flaky search related to iops metrics. ([#35081](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35081))
+- (Contrib) `azuremonitorexporter`: fix issue for property endpoint is ignored when using instrumentation_key ([#33971](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33971))
```
Comment (dmitryax) [inline CHANGELOG.md:49]: I don't think we have this exporter. Please double-check that we include items for components that we provide with our distro.

**Pull request merged**